src/main/java/com/mastery/java/task/dto/validator/IsGender.java

```diff
@@ -0,0 +1,18 @@
+package com.mastery.java.task.dto.validator;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+@Constraint(validatedBy = IsGenderValidator.class)
+@Target( { ElementType.METHOD, ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface IsGender {
+    String message() default "Некоректный пол, коррктное значение - MALE/FEMALE";
```

Comment:
please use english language for notifications

Reply:
done